pkg/api/rest/types.go

```diff
@@ -60,7 +81,7 @@ type podStrategy struct {
 
 // Pods is the default logic that applies when creating and updating Pod
 // objects.
-var Pods RESTCreateStrategy = podStrategy{api.Scheme, api.SimpleNameGenerator}
+var Pods = podStrategy{api.Scheme, api.SimpleNameGenerator}
```

Comment:
I'm open to better names - Creater or Updater might confuse people with apiserver.RESTCreater, but there is a parallel.

----- Original Message -----
> > @@ -60,7 +81,7 @@ type podStrategy struct {
> >  
> >  // Pods is the default logic that applies when creating and updating Pod
> >  // objects.
> > -var Pods RESTCreateStrategy = podStrategy{api.Scheme,
> > api.SimpleNameGenerator}
> > +var Pods = podStrategy{api.Scheme, api.SimpleNameGenerator}
> 
> Is Strategy still the right name for this? It's an ObjectTyper and
> NameGenerator? Why are these separate, anyhow?
> 
> Could we please adopt a convention for declaring what interfaces receivers
> implement? #2992
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/4419/files#r24680126